From 537a1fadc3cb9325e078f405a2e4dd6e51065fd2 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Thu, 12 Nov 2020 18:26:33 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568: comment out bifurcation mode
 for PCIe30

We don't need it and should add this if needed in the evb boards. But
now we leave a comment in rk3568.dtsi to make a reference for BSP
developers.

Change-Id: Idf829f2833dbc8e631b15ecc93baddc72d125d3e
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 6602064a6798..599fc6d77383 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -1502,7 +1502,7 @@
 		reg-names = "pcie-dbi", "pcie-apb";
 		resets = <&cru SRST_PCIE30X1_POWERUP>;
 		reset-names = "pipe";
-		rockchip,bifurcation; /* lane1 when using 1+1 */
+		/* rockchip,bifurcation; lane1 when using 1+1 */
 		status = "disabled";
 	};
 
@@ -1538,7 +1538,7 @@
 		reg-names = "pcie-dbi", "pcie-apb";
 		resets = <&cru SRST_PCIE30X2_POWERUP>;
 		reset-names = "pipe";
-		rockchip,bifurcation; /* lane0 when using 1+1 */
+		/* rockchip,bifurcation; lane0 when using 1+1 */
 		status = "disabled";
 	};
 
-- 
2.35.3

